docs/theme/static/js/components/upcoming-event/upcoming-event.js

```diff
@@ -0,0 +1,71 @@
+/* global moment */
+import $ from 'jquery';
+// import moment from 'moment'; // added globally
```

Comment:
There is but the path isn't very friendly.  http://stealjs.com/docs/StealJS.loading-from-cdn.html